src/containers/DispatchPage/index.js

```diff
@@ -250,6 +252,7 @@ class Page extends Component {
       message.success(`Dispatch "${question.name}" successfully.`);
       // re-subscribe the new record
       this.subscribeRecordUpdate();
+      window.location.reload();
```

Comment:
Should figure out a way not to reload the whole page, it dramatically affects the user experience.